### DISCUSSION TITLE
Allow Right Shift and Right Ctrl Modifiers for Shortcuts

### DISCUSSION THREAD

**Discussion by frustak** [johnlindquist/kit#1414] [Q&A]
Currently, using the right shift key for shortcuts triggers both the left and right shift keys.
Is it possible to use `right shift` or `right ctrl` modifiers for shortcuts?

**Comment by johnlindquist** [comment]
@frustak No, it's not currently possible.

I'll look into adding it for future versions.

To clarify, are you talking about `// Shortcuts: ` at the script level, or `shortcuts: []` as part of the prompt API?

**Reply by frustak** [reply]
Ok thanks, I meant  `// Shortcuts: ` at the script level.
